world/content/riverport/central.ev

```diff
@@ -0,0 +1,353 @@
+#
+# Central area of Riverport
+#
+# start in "Limbo"
+#
+@tel Limbo
+
+######
+# create the first room
+#
+@dig Kai River Bridge;riverport#01 = riverport
+#
+# describe the entrance into the game from limbo
+#
+@desc riverport =
+ This is the entrance to the |mAinneve|n game world.
+#
+# move to the room
+#
+riverport
+#
+# set zone tag
+#
+@tag riverport#01 = riverport : zone
+#
+# set terrain
+#
+@terrain = EASY
+#
+# set range field
+#
+@rangefield = (4, 3)
+#
+# next, set its description
+#
+@desc
+
+ You stand in the middle of an ancient |wbridge|n spanning the |gKai River|n.
+
+ The |wriver|n runs from the north to the south, and a well-traveled |wroad|n
+ runs east and west.
+
```

Comment:
We should document color choices to stay consistent. We also might want to plan for color schemes, where we don't actually put color tags in descs but some other semantic markup.